multiqc/modules/mosaicatcher/mosaicatcher.py

```diff
@@ -0,0 +1,182 @@
+import logging
+from collections import OrderedDict
+
+from multiqc.modules.base_module import BaseMultiqcModule
+from multiqc.plots import bargraph, table
+
+log = logging.getLogger(__name__)
+
+
+class MultiqcModule(BaseMultiqcModule):
+    def __init__(self):
+        super(MultiqcModule, self).__init__(
+            name="MosaiCatcher",
+            anchor="mosaicatcher",
+            href="https://github.com/friendsofstrandseq/mosaicatcher",
+            info=(
+                "Mosaicatcher counts Strand-seq reads and classifies strand states "
+                "of each chromosome in each cell using a Hidden Markov Model."
+            ),
+            doi="10.1038/s41587-019-0366-x",
+        )
+        log_files = self.find_log_files("mosaicatcher", filehandles=False)
+        if not log_files:
+            log.warning("No log files found for MosaiCatcher module")
+        else:
+            print("Log files found:", log_files)
```

Comment:
Should call `self.ignore_samples ` after parsing, and if no log files are found, should raise a `UserWarning` - see [docs](https://multiqc.info/docs/development/modules/#filtering-by-parsed-sample-names) and [example](https://github.com/ewels/MultiQC/blob/master/multiqc/modules/afterqc/afterqc.py#L36-L40)

Reply:
Got it, done!

Reply:
Thanks!